modules/theme-tools/compat/twentytwenty.css

```diff
@@ -0,0 +1,130 @@
+/**
+ * Infinite scroll
+ */
+
+/* Globally hidden elements when Infinite Scroll is supported and in use. */
+.infinite-scroll .pagination-wrapper,
+.infinite-scroll.neverending .footer-nav-widgets-wrapper,
+.infinite-scroll.neverending #site-footer {
+	/* Theme Footer (when set to scrolling) */
+	display: none;
+}
+
+ /* When Infinite Scroll has reached its end we need to re-display elements that were hidden (via .neverending) before. */
+.infinite-end.neverending .footer-nav-widgets-wrapper,
+.infinity-end.neverending #site-footer {
+	display: block;
+}
+
+.infinite-loader {
+	margin: calc(3 * 1rem) auto;
+}
+
+.infinite-loader .spinner {
+	margin: 0 auto;
+	left: inherit !important;
+}
+
+#site-content #infinite-handle {
+	margin: 0 auto;
+	max-width: 58rem;
+	width: calc(100% - 8rem);
+}
+
+#site-content #infinite-handle span {
+	background: transparent;
+	display: block;
+	font-size: 1.7rem;
+	text-align: center;
+}
+
+#site-content #infinite-handle span button,
+#site-content #infinite-handle span button:hover,
+#site-content #infinite-handle span button:focus {
+	-webkit-appearance: none;
+	-moz-appearance: none;
+	background: #cd2653;
```

Comment:
I wonder if we could get this color from the theme. It looks like it's probably the Accent Color Hue based on what I see in the customizer. (I'm guessing this is possibly tricky)

Reply:
Yeah, that bothers me as well. I wanted to grab that accent color but it's not straightforward. I'll try to figure this out.

Reply:
Done in 83aec143345d1447059e294cd8e8c68c2bb1fc10